webapi/_test_data/ontologies/beol-onto.ttl

```diff
@@ -425,6 +442,42 @@
                    rdfs:subPropertyOf knora-base:hasValue .
 
     ### ###########################################
+    ### beol:hasTitle
+
+    :hasTitle rdf:type owl:ObjectProperty ;
```

Comment:
The change to `beol:personHasTitle` would be fine.

I would change `beol:title` myself because I have to coordinate it with my import scripts.